src/flows_dst.rs

```diff
@@ -81,13 +93,15 @@ impl<'a> RespondToRelocateRequests<'a> {
 }
 
 #[derive(Debug, PartialEq)]
-pub struct StartRelocatedNodeConnection<'a>(pub &'a mut MemberState);
+pub struct StartResourceProof<'a>(pub &'a mut MemberState);
 
-impl<'a> StartRelocatedNodeConnection<'a> {
+impl<'a> StartResourceProof<'a> {
     // TODO - remove the `allow` once we have a test for this method.
     #[allow(dead_code)]
     fn start_event_loop(&mut self) {
-        self.schedule_time_out()
+        self.0
+            .action
+            .schedule_event(LocalEvent::CheckResourceProofTimeout);
```

Comment:
This seems to be different than in the diagrams. Should the diagram be updated, then?

Reply:
Yes, there is a mistake in the diagram

Reply:
fixed